microsoft-365/compliance/ediscovery-permissions-filtering-for-content-search.md

```diff
@@ -115,12 +115,16 @@ The  _Filters_ parameter specifies the search criteria for the compliance securi
   > [!IMPORTANT]
   > A single search filter can't contain a mailbox filter and a mailbox content filter. To combine these in a single filter, you have to use a [filters list](#using-a-filters-list-to-combine-filter-types).  But a filter can contain a more complex query of the same type. For example, `"Mailbox_CustomAttribute10 -eq 'FTE' -and Mailbox_MemberOfGroup -eq '$($DG.DistinguishedName)'"`
 
-- **Site and site content filtering:** There are two SharePoint- and OneDrive-related filters that you can use to specify what site or site content the assigned users can search.
+- **Site and site content filtering:** 
 
-  - **Site_**_SearchableSiteProperty_
-  - **SiteContent_**_SearchableSiteProperty_
+**DocumentLink_**_SearchableSiteProperty_
 
-   These two filters are interchangeable. For example, `"Site_Path -like 'https://contoso.sharepoint.com/sites/doctors'"` and  `"SiteContent_Path -like 'https://contoso.sharepoint.com/sites/doctors'"` return the same results. For a list of searchable site properties, see [Keyword queries and search conditions for eDiscovery](ediscovery-keyword-queries-and-search-conditions.md#searchable-site-properties)  For a more complete list, see [Overview of crawled and managed properties in SharePoint](/SharePoint/technical-reference/crawled-and-managed-properties-overview). Properties marked with a **Yes** in the **Queryable** column can be used to create a site or site content filter.
+The path (URL) of a specific folder on a SharePoint or OneDrive site. If you use this property, be sure to search the site where the specified folder is located in. We recommend using this property instead of the Site and Path properties.
+
+To return items located in subfolders of the folder that you specify for the documentlink property, you have to add /* to the URL of the specified folder; `documentlink: "https://contoso.sharepoint.com/Shared Documents/*"`
+
+
+For a list of searchable site properties, see [Keyword queries and search conditions for eDiscovery](ediscovery-keyword-queries-and-search-conditions.md#searchable-site-properties). For a more complete list, see [Overview of crawled and managed properties in SharePoint](/SharePoint/technical-reference/crawled-and-managed-properties-overview). Properties marked with a **Yes** in the **Queryable** column can be used to create a site or site content filter.
```

Comment:
```suggestion
   The path (URL) of a specific folder on a SharePoint or OneDrive site. If you use this property, be sure to search the site where the specified folder is located in. We recommend using this property instead of the Site and Path properties.

   To return items located in subfolders of the folder that you specify for the documentlink property, you have to add /* to the URL of the specified folder; `documentlink: "https://contoso.sharepoint.com/Shared Documents/*"`

   For a list of searchable site properties, see [Keyword queries and search conditions for eDiscovery](ediscovery-keyword-queries-and-search-conditions.md#searchable-site-properties). For a more complete list, see [Overview of crawled and managed properties in SharePoint](/SharePoint/technical-reference/crawled-and-managed-properties-overview). Properties marked with a **Yes** in the **Queryable** column can be used to create a site or site content filter.
```